file-upload/file-upload-java-sbt/build.sbt

```diff
@@ -0,0 +1,36 @@
+organization in ThisBuild := "com.example"
+version in ThisBuild := "1.0-SNAPSHOT"
+
+// the Scala version that will be used for cross-compiled libraries
+scalaVersion in ThisBuild := "2.12.4"
+
+val macwire = "com.softwaremill.macwire" %% "macros" % "2.3.0" % "provided"
```

Comment:
I think `macwire` is not in use since this is Java code.

Reply:
Of course, sorry 😞 
After copy from scala recipe, we removed `macwire` from `libraryDependencies`, but forget remove from variables.